includes/derivatives.inc

```diff
@@ -301,68 +342,17 @@ function islandora_ocr_derive_ocr(AbstractObject $object, $force = FALSE) {
 }
 
 /**
+ * Derive the OCR stream from OBJ.
+ */
+function islandora_ocr_derive_ocr(AbstractObject $object, $force = FALSE) {
```

Comment:
I see, there is a wrapper so maybe the default for type is not really needed

Reply:
Yes, not sure? It was broken out because the original "derive OCR" actually derives both, which doesn't really work if you want to pick which one you're deriving.